templates/search-results.hbs

```diff
@@ -3,113 +3,135 @@
+                </span>
 
-                <img src="/static/images/icon/updated.svg" alt="updated" title="Last Updated"/>
-                <p title="Last Updated">{{this.date_modified}}</p>
+                <span class="mod-updated" title="Last Updated">
+                    <img src="/static/images/icon/updated.svg" alt="updated"/><!--
+                    -->{{this.date_modified}}
+                </span>
 
-                <img src="/static/images/icon/version.svg" alt="version" title="Version"/>
-                <p title="Version">{{this.latest_version}}</p>
+                <span class="mod-version" title="Version">
+                    <img src="/static/images/icon/version.svg" alt="version" /><!--
+                    -->{{this.latest_version}}
+                </span>
 
                 <div class="loader-icons">
                     {{#contains this.keywords "forge"}}
-						<svg viewBox="0 0 120 66.7" class="forge">
-							<path d="M91.6,16.7l-37.8-1.9l46.2,0v-3.7H47.8l0,7.8v6.2c0,0.1-1.5-9.1-1.9-11.7h-4.1v6.8v6.2
-										c0,0.1-1.8-10.9-1.9-12.3c-10.4,0-27.9,0-27.9,0c1.9,1.6,12.4,10.6,19.9,14.3c3.7,1.8,8.3,1.9,12.4,2c2.1,0.1,4.2,0.2,5.8,1.8
-										c2.3,2.2,2.8,5.7,0.8,8.3c-1.9,2.6-7.3,3.2-7.3,3.2L39,49.1v6.4h10.3l0.3-6.3l8.9-6.3c-0.9,0.8-3.1,2.8-6.2,7.7
-										c-0.7,1.1-1.3,2.3-1.7,3.5c2.2-1.9,6.8-3.2,12.2-3.2c5.3,0,9.9,1.3,12.1,3.2c-0.4-1.2-1-2.4-1.7-3.5c-3.2-4.9-5.3-6.9-6.2-7.7
-										l8.9,6.3l0.3,6.3h9.6v-6.4l-4.5-5.5c0,0-6.7-0.4-8.4-3.2C67.7,32.6,74.8,20.4,91.6,16.7z"/>
-						</svg>
+                        <div class="loader-forge" title="Forge">
+                            <svg viewBox="0 0 120 66.7" class="forge">
+                                <path d="M91.6,16.7l-37.8-1.9l46.2,0v-3.7H47.8l0,7.8v6.2c0,0.1-1.5-9.1-1.9-11.7h-4.1v6.8v6.2
+                                            c0,0.1-1.8-10.9-1.9-12.3c-10.4,0-27.9,0-27.9,0c1.9,1.6,12.4,10.6,19.9,14.3c3.7,1.8,8.3,1.9,12.4,2c2.1,0.1,4.2,0.2,5.8,1.8
+                                            c2.3,2.2,2.8,5.7,0.8,8.3c-1.9,2.6-7.3,3.2-7.3,3.2L39,49.1v6.4h10.3l0.3-6.3l8.9-6.3c-0.9,0.8-3.1,2.8-6.2,7.7
+                                            c-0.7,1.1-1.3,2.3-1.7,3.5c2.2-1.9,6.8-3.2,12.2-3.2c5.3,0,9.9,1.3,12.1,3.2c-0.4-1.2-1-2.4-1.7-3.5c-3.2-4.9-5.3-6.9-6.2-7.7
+                                            l8.9,6.3l0.3,6.3h9.6v-6.4l-4.5-5.5c0,0-6.7-0.4-8.4-3.2C67.7,32.6,74.8,20.4,91.6,16.7z"/>
+                            </svg><!--
+                            --><span>Forge</span>
+                        </div>
                     {{/contains}}
                     {{#contains this.keywords "fabric"}}
-                        <img alt="fabric" src="/static/images/icon/fabric.png"/>
+                        <div class="loader-fabric" title="Fabric">
+                            <img alt="fabric" src="/static/images/icon/fabric.png"/><!--
+                            --><span>Fabric</span>
+                        </div>
```

Comment:
The text should be removed IMO. It doesn't add much, as you could see the logo icons already and it looks cluttered

Reply:
I can remove the text, but the icons are very small, so it's hard to understand them if you don't already know the icons for forge and fabric.  Since I added a title text to the icon, it should be fine to remove the text to reduce clutter, but it may reduce readability.

Also, is the forge icon an inline svg just for changing the color with themes?  It makes it harder to align properly, and seems like using a normal image combined with css filters or clip masks would be more efficient.

Reply:
Yeah, the inline svg is only there to change the color. If you have another way to do it, that would be great! It would save a lot of bandwidth

Reply:
I've replaced the inline SVG with a SVG `<use href="#...">` element, so the icon is only inlined once.  For some reason this works with the original theming CSS, so it's only changes to the templates.